src/app/billingentity/billingentity-form/billingentity-form.component.html

```diff
@@ -0,0 +1,107 @@
+<form (submit)="save()" [formGroup]="form" class="flex flex-column p-fluid">
+  <div class="field col-12 mb-3">
+    <label for="displayName" i18n>Display Name</label>
+    <input formControlName="displayName" id="displayName" pInputText type="text"/>
+  </div>
+
+  <p-divider styleClass="col-12">
+    <div class="inline-flex align-items-center">
+      <span class="text-lg font-semibold text-700" i18n>Company</span>
+    </div>
+  </p-divider>
+  <div class="mb-2">
+
+    <div class="field mb-2 col-12">
+      <label for="companyEmail" i18n>Emails</label>
+      <p-chips
+        formControlName="companyEmail"
+        id="companyEmail"
+        separator=","
+        [allowDuplicate]="false"
+        placeholder="Enter comma separated"
```

Comment:
```suggestion
```

I would like to make it less obvious that email is a multi value field.

Furthermore, the decoration behavior is confusing and not consistent. Also decorate the address with the nice bubble and remove marker when:

* entering a space
* moving the cursor to another field (tab mouse or otherwise)

The decoration currently only happens when entering a `,`. While adding the behavior for space, keep the one for `,`.

Reply:
It should be possible to achieve what you want.

Although there's a bug in the input component used (https://github.com/primefaces/primeng/issues/12640), I could make it work with a workaround.